### PR TITLE
Add workflow for weblate template update

### PR DESCRIPTION
**Pull request by cschramm**

### CODE REVIEW

File: .github/workflows/weblate.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - weblate
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo apt-get install intltool curl
+      - run: intltool-update --pot
```
Comment (infirit) [inline .github/workflows/weblate.yml:16]: I just dropped intltool so I'll merge this one first and then update the workflow.

**Comment by cschramm** [comment]
Hrm, it looks like this constantly results in updated timestamps in the po files from Weblate, see https://github.com/blueman-project/blueman/pull/1233/commits/148b44556441a388e906047859bcc5701a77d89a... :confused:

**Comment by infirit** [comment]
weblate appears hosted in germany so maybe a timezone thing?

**Comment by cschramm** [comment]
I think the point is that when the job generates the pot file, it adds the current time as `POT-Creation-Date` and Weblate seems to spread that into all po files even if nothing changed. To avoid that, we'd have to check if something changed before uploading. A straightforward solution would be to leave that to gettext by first downloading Weblate's pot file. I think gettext will only touch the timestamp if something actually changed, so that otherwise we'll upload the exact same file that we just downloaded.

**Pull request merged**